templates/includes/delete_confirm.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+
+{% block content %}
+  {% block h1 %}
+  {% endblock %}
+  <form action="" method="post">
+    {% csrf_token %}
+
+    <input type="submit" value="Yes" class="btn btn-danger">
+  </form>

```

Comment:
You should have a `Cancel` button apart from the `Delete` one. The `Cancel` button will lead to the previous page the user was on.